bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -210,6 +210,7 @@
         "auditorMaxNumberOfConcurrentOpenLedgerOperations";
     protected static final String AUDITOR_ACQUIRE_CONCURRENT_OPEN_LEDGER_OPERATIONS_TIMEOUT_MSEC =
         "auditorAcquireConcurrentOpenLedgerOperationsTimeOutMSec";
+    protected static final String REPLICATION_RATE_BY_BYTES = "replicationRateByBytes";
```

Comment:
isn't it a ClientConfiguration ? why we need the property also on ServerConfiguration ?

Reply:
We need config it through server config file.